samples/batch_fetch_test.rb

```diff
@@ -11,7 +11,8 @@
 )
 
 batch_fetch_request = {
-  :filters => "BatchId=2956681"
+  :fields => "Files",
+  :filters => "BatchId=3009916"
```

Comment:
I just don't think the ID should change with each new commit. Ideally, it would be configurable or passed in through command line.